docs/source/python/data.rst

```diff
@@ -518,17 +518,15 @@ Record Batch Readers
 
 Many functions in PyArrow either return or take as an argument a :class:`RecordBatchReader`.
 It can be used like any iterable of record batches, but also provides their common
-schema without having to get any of the batches.
-
-.. ipython:: python
-
-   schema = pa.schema([('x', pa.int64())])
-   def iter_record_batches():
-      for i in range(2):
-         yield pa.RecordBatch.from_arrays([pa.array([1, 2, 3])], schema=schema)
-   reader = pa.RecordBatchReader.from_batches(schema, iter_record_batches())
-   print(reader.schema)
-   for batch in reader:
-      print(batch)
+schema without having to get any of the batches.::
+
+   >>> schema = pa.schema([('x', pa.int64())])
+   >>> def iter_record_batches():
+   ...    for i in range(2):
+   ...       yield pa.RecordBatch.from_arrays([pa.array([1, 2, 3])], schema=schema)
+   >>> reader = pa.RecordBatchReader.from_batches(schema, iter_record_batches())
+   >>> print(reader.schema)
```

Comment:
You can add the output here of the print statement (so users see what the code does)

Reply:
Thanks! Sorry I forgot that.